dff/stats/cli.py

```diff
@@ -194,7 +198,8 @@ def make_zip_config(parsed_args: argparse.Namespace) -> Path:
     if OmegaConf.select(cli_conf, "db.driver") == "clickhousedb+connect":
         params = dict(
             table="${db.table}",
-            lag="neighbor(label, -1)",
+            label_lag="(request_id != '0' ? neighbor(label, -1) : '')",
+            flow_lag="(request_id != '0' ? neighbor(flow_label, -1) : '')",
```

Comment:
Apparently, there's an issue with using `neighbor` with `GROUP BY`.

If I run the query without `GROUP BY`, then it returns correct `prev_label` values.
But with `GROUP BY` it returns values like these:
![image](https://github.com/deeppavlov/dialog_flow_framework/assets/61429541/02d4aafb-2e7f-4704-9dc2-641adf88916c)

This [page](https://kb.altinity.com/altinity-kb-queries-and-syntax/lag-lead/) shows various ways to implement the previous item retrieval and the section with `neighbor` is titled "Using neighbor (no grouping, incorrect result over blocks)", so it seems like `neighbor` should not be used with `GROUP BY` (although I couldn't find github issues about it).

Another approach suggested in the page works though:
```suggestion
            label_lag="lagInFrame(label) OVER "
                      "(PARTITION BY context_id ORDER BY request_id ASC "
                      "ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)",
            flow_lag="lagInFrame(flow_label) OVER "
                     "(PARTITION BY context_id ORDER BY request_id ASC "
                     "ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING)",
```